packages/contracts/contracts/curation/Curation.sol

```diff
@@ -113,11 +113,9 @@ contract Curation is CurationV2Storage, GraphUpgradeable {
      * @notice Update the minimum deposit amount to `_minimumCurationDeposit`
      * @param _minimumCurationDeposit Minimum amount of tokens required deposit
      */
-    function setMinimumCurationDeposit(uint256 _minimumCurationDeposit)
-        external
-        override
-        onlyGovernor
-    {
+    function setMinimumCurationDeposit(
```

Comment:
hmm is it possible that the linter config broke with the move? seems it's changing everything to a different format, but the .prettierrc.json hasn't changed, so maybe it's not picking it up anymore?

Reply:
mmm looks like you are right. fixing.

Reply:
Ok found the root cause. When converting to a monorepo I deleted yarn lock file to bump dependencies, it looks like newer versions of our linting tools just have different rules. 

If you delete `yarn.lock` on the `main` branch and do a fresh install then lint the code it will output the code linted as in this PR. 

So I'm inclined to keep this as is.